doc/ref/configuration/minion.rst

```diff
@@ -752,6 +752,30 @@ Statically assigns grains to the minion.
       cabinet: 13
       cab_u: 14-15
 
+.. conf_minion:: grains_blacklist
+
+``grains_blacklist``
+----------
+
+Default: ``[]``
+
+If :conf_minion:`grains_blacklist` is enabled then any core grains will be
+checked against the matches found in the `grains_blacklist` list and any
+matched core grains will be filtered.
+
+.. note::
+
+    Some modules may depends on core grains. Filtering may cause
+    the dependent module to run unusual.
+
+.. versionadded:: 2018.11.0
```

Comment:
The version for this should be `Neon`. We haven't decided on a release number for the upcoming `Fluorine` release, but we've already passed the feature freeze for it anyway.